### PR TITLE
editoast: add a osm-to-railjson command

### PR DESCRIPTION
**Pull request by Tristramg**
This command will extract data from OpenStreetMap (.pbf file format) and generate the track_sections from Ways that represent rails

It uses the osm4routing crate to cut the ways into edges suitable for routing

ref #4035

### CODE REVIEW

File: editoast/src/main.rs [hunk 1]
```diff
@@ -358,6 +356,53 @@ async fn clear(
...
 }
 
+/// Run the osm-to-railjson subcommand
+/// Converts OpenStreetMap pbf file to railjson
+fn osm_to_railjson(args: OsmToRailjsonArgs) -> Result<(), Box<dyn Error + Send + Sync>> {
+    println!(
+        "🗺️ Converting {} to {}",
+        args.osm_pbf_in.display(),
+        args.railjson_out.display()
+    );
```
Comment (younesschrifi) [inline editoast/src/main.rs]: Add a test to this function

Comment (Tristramg) [reply editoast/src/main.rs]: I’ve added a minimal fixture to test this function

File: editoast/src/client/mod.rs [hunk 2]
```diff
@@ -95,6 +96,15 @@ pub struct ImportProfileSetArgs {
     pub electrical_profile_set_path: PathBuf,
 }
 
+#[derive(Args, Debug)]
+#[command(about, long_about = "Add a set of electrical profiles")]
```
Comment (Khoyo) [inline editoast/src/client/mod.rs]: I'm not sure it does that ;)

Comment (Tristramg) [reply editoast/src/client/mod.rs]: yikes, indeed, thank you!

File: editoast/src/main.rs [hunk 3]
```diff
@@ -359,12 +357,59 @@ async fn clear(
...
+            let c = *counts.entry(e.id.0).or_insert(0);
+            counts.insert(e.id.0, c + 1);
+            schema::TrackSection {
+                id: schema::utils::Identifier(format!("{}-{}", e.id.0, c)),
+                length: e.length(),
+                geo: geos::geojson::Geometry::new(geos::geojson::Value::LineString(
+                    e.geometry.iter().map(|c| vec![c.lon, c.lat]).collect(),
+                )),
+                slopes: vec![],
+                curves: vec![],
```
Comment (flomonster) [inline editoast/src/main.rs]:
```suggestion
```

**Review by flomonster — COMMENTED** [review]
Thanks for your PR! :tada: 
It looks good to me, can you just rewrite your git history to follow up the git commit style describe [here](https://osrd.fr/en/docs/guides/contribute/code/#git-commit-style).

**Review by flomonster — APPROVED** [review]

**Pull request merged**